MVP Module/Source/Feed Presentation/ErrorViewModel.swift

```diff
@@ -0,0 +1,3 @@
+struct ErrorViewModel {
+	let isVisible: Bool
+}
```

Comment:
Ideally, the presentation layer should decide which string to use, including localization logic. This way, the controller or view doesn't need to contain presentation logic.

So instead of a boolean, it could be an optional string to represent both cases (visible or not).

```suggestion
struct ErrorViewModel {
	let message: String?
}
```